src/Microsoft.AspNetCore.Authentication/TokenExtensions.cs

```diff
@@ -63,6 +63,26 @@ public static string GetTokenValue(this AuthenticationProperties properties, str
                 : null;
         }
 
+        public static bool UpdateTokenValue(this AuthenticationProperties properties, string tokenName, string tokenValue)
```

Comment:
We don't really want to encourage that given the way things are setup, since we basically have to nuke the digest key.  Ideally we want them to figure out all their tokens ahead of time and call Store once